www_src/main.less

```diff
@@ -56,6 +58,25 @@ code, pre {
   }
 }
 
+// Screen readers only
```

Comment:
are we using bootstrap? because that's a very bootstrap-specific thing, not generate browser behaviour?